docs/clients/promtail/troubleshooting.md

```diff
@@ -3,6 +3,53 @@
 This document describes known failure modes of `promtail` on edge cases and the
 adopted trade-offs.
 
+## Pipe data to Promtail
+
+Promtail supports piping data for sending logs to Loki. This is a very useful way to troubleshooting your configuration.
```

Comment:
@slim-bean Feel free to reword here. Also should this gets its own page ? I think for now having it in troubleshooting is fine.